core/build.sbt

```diff
@@ -19,7 +19,7 @@ libraryDependencies ++= Seq(
   "org.scalaz"         %% "scalaz-core"    % "7.1.0",
   "org.scalaz.stream"  %% "scalaz-stream"  % "0.6a",
   "org.apache.commons" % "commons-pool2"   % "2.2",
-  "io.netty"           % "netty-transport" % "4.0.25.Final",
+  "io.netty"           % "netty-all" % "4.0.25.Final",
```

Comment:
bunch 'o stuffs. I'll figure out the more fine grained imports, this was a moment of weakness...